src/foam/u2/wizard/axiom/WizardAction.js

```diff
@@ -26,5 +26,19 @@ foam.CLASS({
         return data$canGoNext && ! isLoading_;
       }
     }
+  ],
+  methods: [
+    function toE(args, X) {
```

Comment:
How is this different than the regular Action.toE()?

Reply:
doesn't require the viewspec to already have the `action` property bound to it - we could also just make this change in ActionViewRefinement instead, should I do that?